stable/ambassador/values.yaml

```diff
@@ -216,15 +216,19 @@ pro:
   enabled: false
   image:
     repository: quay.io/datawire/ambassador_pro
-    tag: amb-sidecar-0.5.0
+    tag: amb-sidecar-0.6.0
+  # As of Ambassador Pro 0.6.0, both the RateLimitService and AuthService use the same port
   ports:
     auth: 8500
-    ratelimit: 8501
-    ratelimitDebug: 8502
+    ratelimit: 8500
   logLevel: info
   licenseKey:
     value:
     secret: false
+  # Ambassador Pro environment variables can be found at https://www.getambassador.io/reference/pro/environment
+  # For consistency, AMBASSADOR_ID is copied over from the Ambassador env above and will be ignored if set here.
+  env:
```

Comment:
should probably default to `env: {}`

Reply:
Just to make it clear that the value is an array?

I left it the way it is since I got confused once or twice when `.Values.env` defaulted to `{}`. I assumed the braces were necessary when they are not so I thought omitting them was a cleaner UX.

Reply:
`hasKey` requires `{}` so reverted back to using empty interface notation